streamalert/alert_processor/outputs/credentials/provider.py

```diff
@@ -122,7 +124,7 @@ def save_credentials(self, descriptor, kms_key_alias, props):
                  for (name, prop) in props.items() if prop.cred_requirement}
 
         credentials = Credentials(creds, False, self._region)
-        return self._core_driver.save_credentials_into_s3(descriptor, credentials, kms_key_alias)
+        return self._core_driver.save_credentials(descriptor, credentials, kms_key_alias)
```

Comment:
just so I'm understanding properly, we're going to continue using the old kms key for secrets for the `SecureString` option of ssm right? I think that is the case but just clarifying

Reply:
Yes, so the secrets in SSM will be encrypted with the same KMS key at rest :)

Reply:
okay - just checking if we were creating a new key or not for this